src/streaming/constants/Constants.js

```diff
@@ -246,6 +246,22 @@ export default {
         EVENT: 'event'
     },
 
+    /**
+     *  @constant {string} CMCD_EVENTS specifies all the available events for CMCD.
+     *  @memberof Constants#
+     *  @static
+     */
+    CMCD_EVENTS: {
```

Comment:
What do you think to change this to CMCD_REPORTING_EVENTS or something similar?

```suggestion
    CMCD_REPORTING_EVENTS:
```

Reply:
Renamed CMCD_EVENTS to CMCD_REPORTING_EVENTS